package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "textract",
-  "version": "1.3.0",
```

Comment:
mind not changing this? 1.3.0 isn't released yet, will be next release.  My bad for not having it on a branch considering how long its taken me to dedicate time to it.